includes/class-sensei-teacher.php

```diff
@@ -935,6 +937,8 @@ public function notify_admin_teacher_course_creation( $new_status, $old_status,
 
         // Send mail
         Sensei()->emails->send( $recipient, $subject , Sensei()->emails->get_content( $template ) );
+				
+				do_action('sensei_after_sending_email');
```

Comment:
sorry, but I guess it is not impossible: this action have to be called before any gettext function ( like __() / _e() ). If you want to move ti send(), you will have to move also code responsible for creation of $subject and email body.